finals-2024-01-16/code/5040-dara-poposka/src/app/navigation-bar-component/navigation-bar-component.component.ts

```diff
@@ -0,0 +1,28 @@
+// components/navigation-bar.component.ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-navigation-bar',
```

Comment:
This component is not used anywhere and moreover, it does not make sense for the same component to be both a header and a footer. +5 points total for both, as well as the `nav` and `footer` usage